dockstore-common/src/main/java/io/dockstore/common/yaml/AbstractYamlService.java

```diff
@@ -36,6 +36,10 @@
      * The service's description
      */
     private String description;
+    /**
+     * Change the service's publish-state, if set
+     */
+    private Boolean publish;
```

Comment:
Forget the rules in yaml classes, but is there any value in Boolean as opposed to boolean? 
Do we want to handle null? (or is that mandatory because the key might be missing?)

Reply:
Boolean can be `null`, boolean cannot.
Since I've made this an optional YML field, we want to handle `null.`